packages/api/test/e2e/chain.test.js

```diff
@@ -2,8 +2,8 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
-import Api from '../../api/src';
-import Ws from '../src/ws';
+import Ws from '../../../api-provider/src/ws';
```

Comment:
Could use `@polkadot/provider`? At least it's robust if we change folder names.

Reply:
Yeap, should really do. Will fix.